### ISSUE TITLE
Text associated with the radio group (lable) is not announced by TalkBack in Mobile browser 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5844]
**_Specs: Android
Screen Readers Used: Talkback
Browsers Tested: Google Chrome_**

**Observation:** 
When navigating a radio group “Are you applying as an individual or on behalf of an organization?” inside a Mobile browser using Android TalkBack, the descriptive text (e.g., group label or individual option labels) is not announced. This creates confusion for screen reader users, who may be unaware of the purpose of the radio buttons or which options are available. 

**_Compounded Severity: Critical_** 

**Recommendation:**  
Please see that the next radio group, “Are you applying from within Canada?” is working as expected. 

Verify the issue and implement the same solution for the radio group in question. 

Validate Mobile browser configuration to allow accessibility services to function correctly.
